source/installation-guide/securing_api.rst

```diff
@@ -5,97 +5,76 @@
 Securing the Wazuh API
 ======================
 
-By default, the communication between the Wazuh Kibana plugin and the Wazuh API is not encrypted. In addition to this, the default user is "foo" and the password is  "bar". For this reason, it is highly recommended that you secure the Wazuh API and change the default user and password. The following steps will help you to achieve it:
+By default, the communication between the Wazuh Kibana plugin and the Wazuh API is  encrypted with HTTPS. This means that if you do not provide your own private key and certificate, the Wazuh API will generate its own. In addition to this, both default user and password is ``wazuh``. For this reason, it is highly recommended that you change the default password at least and use your own certificate since the one created by the Wazuh API is self signed. These are the **recommended mininum changes** that you should do:
```

Comment:
```suggestion
By default, the communication between the Wazuh Kibana plugin and the Wazuh API is encrypted with HTTPS. This means that if you do not provide your own private key and certificate, the Wazuh API will generate its own in the first run. In addition, the default user is ``wazuh`` and the default password is also ``wazuh``. For this reason, it is highly recommended to at least change the default password and to use your own certificate since the one created by the Wazuh API is self signed. These are the **recommended changes** to apply:
```